packages/cf-layout/src/organisms/featured-content-module.less

```diff
@@ -19,25 +19,31 @@
         padding-top: unit( @grid_gutter-width / @base-font-size-px, em );
         padding-bottom: unit( @grid_gutter-width / @base-font-size-px, em );
 
+        .respond-to-max( @bp-xs-max, {
+            padding-right: unit( @grid_gutter-width / 2 / @base-font-size-px, em );
+            padding-left: unit( @grid_gutter-width / 2 / @base-font-size-px, em );
+        } );
+
         .respond-to-min( @fcm-bp-min, {
-            padding-right: unit( @grid_gutter-width / @base-font-size-px, em );
+            flex: 1;
+            padding-right: 300px;
```

Comment:
Just checking, fixed padding here is correct?

Reply:
It is, but it should be calculated from variables. Updated in the forthcoming commit.